solc/CommandLineInterface.cpp

```diff
@@ -877,11 +877,13 @@ void CommandLineInterface::handleCombinedJSON()
 	if (m_options.compiler.combinedJsonRequests->ast)
 	{
 		output[g_strSources] = Json::Value(Json::objectValue);
+		unsigned sourceIndex = 0;
 		for (auto const& sourceCode: m_fileReader.sourceUnits())
 		{
 			ASTJsonExporter converter(m_compiler->state(), m_compiler->sourceIndices());
 			output[g_strSources][sourceCode.first] = Json::Value(Json::objectValue);
-			output[g_strSources][sourceCode.first]["AST"] = converter.toJson(m_compiler->ast(sourceCode.first));
+			output[g_strSources][sourceCode.first]["ast"] = converter.toJson(m_compiler->ast(sourceCode.first));
+			output[g_strSources][sourceCode.first]["id"] = sourceIndex++;
```

Comment:
See https://github.com/ethereum/solidity/blob/0cb279494a9af0938dfbf2ca5d0b21115198e8bb/libsolidity/interface/StandardCompiler.cpp#L1300

Reply:
We never had the "id" here before. But because we just add this information now, I guess it's a non-breaking change.